docs/getting-started/getting-started.md

```diff
@@ -433,15 +433,15 @@ You can see that the UI now shows the movies that are stored in the native appli
 
 ## Adding the Navigation API
 
-We created this API for you for this tutorial--you can view it in the GitHub repository located here: [this GitHub repository](https://github.com/electrode-io/react-native-ernnavigation-api). This very simple API will be used for navigating from the `MovieListMiniApp` to the `MovieDetailsMiniApp`.
+We generated this API for you for this tutorial and the code is hosted at [react-native-ernnavigation-api](https://github.com/electrode-io/react-native-ernnavigation-api) repo. This API will be used for navigating from the `MovieListMiniApp` to the `MovieDetailsMiniApp`.
```

Comment:
Can you revert this edit.
Let's keep `created` instead of `generated`. `create-api` is the command. The fact that it's generated is a technical detail of how it gets created. So yes let's stick to the higher level simpler term in the getting started.
Also let's keep the `GitHub repository located here`. Not too big fan of `view` but anyway the first phrase is better, also without abbreviation `repo` vs `repository`. we shouldn't use too much abbr. in a getting started.
this was edited by Catherine I think, there were good reasons behind this wording for the most part ;)

Reply:
Still .. in the original version :`in the GitHub repository located here: [this GitHub repository](https://github.com/electrode-io/react-native-ernnavigation-api)`

That's not right, just noticed it ... should rather be `in the [NavigationApi GitHub repository](https://github.com/electrode-io/react-native-ernnavigation-api)`